src/Plugin/Block/EuFooterBlock.php

```diff
@@ -39,7 +39,7 @@ public function build() {
     }
 
     NestedArray::setValue($build, ['#corporate_footer'], $data);
-    $this->setSiteSpecificFooter($build, $cache);
+    $this->setSiteSpecificFooter($build, $cache, ['ec_core_column']);
```

Comment:
No, let's not add the new column to EU footer because the links there are different and for now only the EC footer should be changed.